tiledb/bioimg/converters/png.py

```diff
@@ -109,7 +109,10 @@ def level_shape(self, level: int = 0) -> Union[Tuple[int, ...], Any]:
                 "PNG images are not pyramidal and consist of only one level"
             )
         # Even after converting to RGB the size is not updated to 3d from 2d
-        l_shape = (*self._png.size, 3)
+        w, h = self._png.size
+
+        # Numpy shape is of the format (H, W, D) compared to Pillow (W, H, D)
+        l_shape = (h, w, 3)
```

Comment:
I think this might still cause problems. I believe png supports 1-4 channels depending on the file. CZI definitely was importing some RGBA data.

Reply:
If transparency is relevant and needed I can definitely modify the code to support it.

Reply:
I think it will be necessary on the SOMA side.